MapboxDirections.xcodeproj/project.pbxproj

```diff
@@ -15,6 +15,8 @@
 		358D48871E2EAB4500F32A65 /* MBRouteOptions.m in Sources */ = {isa = PBXBuildFile; fileRef = 358D48851E2EAB4500F32A65 /* MBRouteOptions.m */; };
 		358D48881E2EAB4500F32A65 /* MBRouteOptions.m in Sources */ = {isa = PBXBuildFile; fileRef = 358D48851E2EAB4500F32A65 /* MBRouteOptions.m */; };
 		358D48891E2EAB4500F32A65 /* MBRouteOptions.m in Sources */ = {isa = PBXBuildFile; fileRef = 358D48851E2EAB4500F32A65 /* MBRouteOptions.m */; };
+		C51538CC1E807FF00093FF3E /* MBAnnotation.swift in Sources */ = {isa = PBXBuildFile; fileRef = C51538CB1E807FF00093FF3E /* MBAnnotation.swift */; };
```

Comment:
This file needs to be added to all four framework targets. Because it’s only in the iOS framework target, macOS, tvOS, and watchOS builds are failing with numerous errors, starting with:

```
▸ Compiling MBRouteLeg.swift

❌  /Users/vagrant/git/MapboxDirections/MBRouteLeg.swift:11:36: use of undeclared type 'SegmentAttribute'

    typealias SegmentAttributes = [SegmentAttribute:[Any]]
                                   ^~~~~~~~~~~~~~~~



❌  /Users/vagrant/git/MapboxDirections/MBRouteLeg.swift:129:34: use of undeclared type 'SegmentAttribute'

    open let segmentAttributes: [SegmentAttribute:[Any]]?
                                 ^~~~~~~~~~~~~~~~



❌  /Users/vagrant/git/MapboxDirections/MBRouteLeg.swift:173:14: cannot invoke 'RouteLegV4.init' with an argument list of type '(steps: [RouteStepV4], json: JSONDictionary, source: Waypoint, destination: Waypoint, profileIdentifier: MBDirectionsProfileIdentifier)'

        self.init(steps: steps, json: json, source: source, destination: destination, profileIdentifier: profileIdentifier)
             ^
```